docs/source/troubleshooting/troubleshooting-getting-started.adoc

```diff
@@ -53,6 +53,12 @@ You can also add this variable in your shell profile so you don't need to manual
 While {project} starts, it runs several startup checks to make sure that the {project} VM and the OpenShift Cluster are able to start without any issues.
 If any configuration is incorrect or missing, the startup checks fail and {project} does not start.
 
+- You can skip the startup checks by executing the following command:
```

Comment:
There's no reason for this to get in the way of the PR being merged, but is there a reason that this is in a bulleted list? Just curious. :)

Reply:
https://docs.openshift.org/latest/minishift/troubleshooting/troubleshooting-getting-started.html#minshift-startup-check-failed If you see other flags related to startup are mentioned in bullet list so added that way

Reply:
But if you think, we should not put in the bullet list. I can change.

Reply:
I like that it's consistent, it just surprised me a little bit because I wouldn't have written things that way. It's absolutely fine as it is. :+1:

Thank you for clarifying!